### DISCUSSION TITLE
Onion OS has no hostname on the MM+

### DISCUSSION THREAD

**Discussion by AKiniyalocts** [OnionUI/Onion#1025] [Ideas]
Hello all, I am investigating creating an automatic network based backup for my MM+, to backup saves and save states. In order to do this, I need my MM+ to have have a hostname I can refer to on my local network (an IP address wont cut it as that will eventually change).

Currently the MM+ has no hostname:
```
/ # hostname
(none)
```
The above was taken from SSH'ing into my MM+. Normally you are able to set the hostname by editing `/etc/hostname` and `/etc/hosts`. I'm unable to do this from SSH or via terminal on my MM+ since it is a read-only file system. Is this possible to do on the MM+? Would it be possible for Onion to declare a default hostname for the MM+ ?

Thank you!

**Comment by XK9274** [comment]
Neither `/etc/hosts` or `/etc/hostname` exist so we can't even mount them. You can force windows to recognise any hostname for the IP address by editing `C:\Windows\System32\drivers\etc\hosts` but this possibly won't achieve what you want it too. 

I don't believe this will be possible.

**Reply by AKiniyalocts** [reply]
I see. No worries! TBH this would be better performed from the Onion client to the remote/local network server. I just don't know enough C to make it happen 😬

**Reply by XK9274** [reply]
I'd recommend checking out Syncthing:

https://github.com/XK9274/syncthing-app-miyoo

Install on MMP and on PC, choose which folders to keep synced - done. (This also suffers from the hostname issue but there's instructions on there to change this)

```
Device name will be (none) as the hostname never gets set. You can set this by changing the first line containing (none) in the config.xml file in the Syncthing App folder (and then restart)
```
